views/install.sh.erb

```diff
@@ -171,6 +171,13 @@ elif test "x$os" = "xAIX"; then
   platform="aix"
   platform_version="`uname -v`.`uname -r`"
   machine="powerpc"
+elif test -f "/etc/os-release"; then
+  . /etc/os-release
+  platform=$ID
+  if test "x$platform" = "x"; then
+    platform="linux"
+  fi
```

Comment:
that's a bit useless.  if we get "linux" back there's no artifact to serve.  we need 'debian' or 'ubuntu' or 'el' or whatever.